django/forms/utils.py

```diff
@@ -49,12 +51,17 @@ def get_context(self):
         )
 
     def render(self, template_name=None, context=None, renderer=None):
-        return mark_safe(
-            (renderer or self.renderer).render(
-                template_name or self.template_name,
-                context or self.get_context(),
+        renderer = renderer or self.renderer
+        template = template_name or self.template_name
+        if template == "django/forms/default.html":
+            msg = (
+                'The template "django/forms/default.html" will be removed.'
+                "You should specify an appropriate template, such as "
+                '"django/forms/div.html" on your FORM_RENDERER, or Form or '
+                "FormSet subclass."
```

Comment:
Can this be clearer? Do we want to refer to `form_template_name`/`template_name`?